src/core/server/app/middleware/logging.ts

```diff
@@ -37,7 +37,7 @@ export const access: RequestHandler = (req, res, next) => {
   next();
 };
 
-export const error: ErrorRequestHandler = (err, req, res, next) => {
+export const error: ErrorRequestHandler = (err, _req, _res, next) => {
```

Comment:
What's the benefit of this rule for unused params if it just means that to fix bugs with airity, we have to prefix all our params with `_`?

Reply:
I think we can turn it off, also not sure about its benefits.